02_activities/assignments/assignment_2.ipynb

```diff
@@ -83,12 +97,12 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Your answer here"
+    "Unfortunately, my partner's solutions doesn't work for all examples. Consider the following counter example.\n",
+    "\n",
+    "If $l = [1,1]$, then `missing_num(l)` returns `[0]`. The correct return value is `[0,2]`."
```

Comment:
Actually I wounder how did you come to the point, that missing values in your examples are 0 and 2?